src/dataflow/render/mod.rs

```diff
@@ -681,8 +716,10 @@ where
 
                 RelationExpr::Negate { input } => {
                     self.ensure_rendered(input, env, scope, worker_index);
-                    let collection = self.collection(input).unwrap().negate();
-                    self.collections.insert(relation_expr.clone(), collection);
+                    let (ok_collection, err_collection) = self.collection(input).unwrap();
+                    let ok_collection = ok_collection.negate();
+                    self.collections
+                        .insert(relation_expr.clone(), (ok_collection, err_collection));
```

Comment:
Maybe weird, but it might be best to negate the `err_collection` too. Back up in `server.rs`, it will yell if it sees a negative error accumulation, and it seems possible (not sure how) that not maintaining the signs faithfully could result in a negation.

Or, it might be really important never to negate the error collection, and then we are sure not to have that problem... Worth discussing / talking through?

Reply:
IIRC we talked about this offline and decided the current behavior is good?